cephbackupstorage/cephbackupstorage/cephagent.py

```
@@ -1072,6 +1121,138 @@ def cancel(self, req):
         rsp = AgentResponse()
         return jsonobject.dumps(plugin.cancel_job(cmd, rsp))
 
+    @replyerror
+    def download_file(self, req):
+        rsp = DownloadFileRsp()
+        cmd = jsonobject.loads(req[http.REQUEST_BODY])
+
+        # 获取 file 大小，如果目录容量不足了（需要两倍空间），直接报错
```

Ruff (0.14.10)

Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)